templates/synthetics.config.ts

```diff
@@ -0,0 +1,16 @@
+import type { SyntheticsConfig } from '@elastic/synthetics';
+
+export default env => {
+  const config: SyntheticsConfig = {
+    params: {
+      url: 'https://elastic.github.io/synthetics-demo/',
+    },
+    playwrightOptions: {
+      ignoreHTTPSErrors: true,
+    },
+  };
+  if (env !== 'development') {
+    config.params.url = 'https://elastic.github.io/synthetics-demo/';
+  }
```

Comment:
Not sure we need this in the template.

Reply:
Thought it would be helpful for users to find it easier to override params if desired.

Reply:
Hmm, I'm not strongly opinionated about that, but I'm fine to keep it. If we end up going with the `example.com` environment then we can just change this to `example.org` and have the default URL being `example.com` I guess.